core/src/epicli/data/common/ansible/playbooks/roles/postgresql/tasks/upgrade/extensions/replication/pg-new/repmgr-conf.yml

```diff
@@ -20,3 +20,10 @@
     _replication_user_name: "{{ specification.extensions.replication.replication_user_name }}"
     _repmgr_database: "{{ specification.extensions.replication.repmgr_database }}"
     _repmgr_service_name: "{{ new_version.repmgr.service_name[ansible_os_family] }}"
+
+- name: PostgreSQL | Extensions | repmgr | Ubuntu | Update symlink to config file
```

Comment:
In all other places there is "Debian" instead of "Ubuntu" in task names. I suggest change a task name here or in other places.

Reply:
I would vote for the second option since we are supporting only Ubuntu, not other Debian-based distros.

Reply:
Updated to "Debian" for consistency with other places and condition `when: ansible_os_family == 'Debian'`.